endpoints-support.md

```diff
@@ -366,6 +366,23 @@
+|<sub>/rest/users/administrator/resetPassword</sub>                                        | PUT      |  :heavy_multiplication_x:   | :heavy_multiplication_x:   | :heavy_multiplication_x:   |  :heavy_multiplication_x:   |  :heavy_multiplication_x:   | :heavy_multiplication_x:   | :heavy_multiplication_x:   | :heavy_multiplication_x:   | :heavy_multiplication_x:   | :heavy_multiplication_x:  | :heavy_multiplication_x:   | :heavy_multiplication_x:   |
+|<sub>/rest/users/changePassword</sub>                                                     | POST      |  :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |  :heavy_minus_sign:   |  :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |  :heavy_minus_sign:   |  :heavy_minus_sign:   | :white_check_mark:  |
+|<sub>/rest/users/roles</sub>                                                              | DELETE      |  :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |  :heavy_minus_sign:   |  :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |  :heavy_minus_sign:   |  :heavy_minus_sign:   | :white_check_mark:  |
+|<sub>/rest/users/roles/users/{role}</sub>                                                 | GET      |  :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |  :heavy_minus_sign:   |  :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |  :heavy_minus_sign:   |  :heavy_minus_sign:   | :white_check_mark:  |
+|<sub>/rest/users/{userName}</sub>                                                         | GET      |  :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |  :heavy_minus_sign:   |  :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |  :heavy_minus_sign:   |  :heavy_minus_sign:   | :white_check_mark:  |
+|<sub>/rest/users/{userName}</sub>                                                         | DELETE      |  :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |  :heavy_minus_sign:   |  :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |  :heavy_minus_sign:   |  :heavy_minus_sign:   | :white_check_mark:  |
+|<sub>/rest/users/{userName}/roles</sub>                                                         | POST      |  :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |  :heavy_minus_sign:   |  :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |  :heavy_minus_sign:   |  :heavy_minus_sign:   | :white_check_mark:  |
```

Comment:
you have kept heavy_minus_sign for all the API Versions except v2800, 
heavy_minus_version means Endpoint not available for this API Version.
I don't think that this resource cant be used for API Version lesser than v2800, please check and confirm.